### PR TITLE
1.23.0

### PR DESCRIPTION
**Pull request by denobot**
Bumped versions for 1.23.0

Please ensure:
- [x] Target branch is correct
- [x] Crate versions are bumped correctly
- [x] deno_std version is incremented in the code (see `cli/compat/mod.rs`)
- [ ] Releases.md is updated correctly

To make edits to this PR:
```shell
git fetch upstream release_1_23.0 && git checkout -b release_1_23.0 upstream/release_1_23.0
```

cc @aslilac

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.23.0 / 2022.06.15
+
+- BREAKING: remove `Intl.v8BreakIterator` (#14864)
+- Deno.exit() is an alias to self.close() in worker contexts (#14826)
+- Remove .github/CODEOWNERS (#14781)
```
Comment (dsherret) [inline Releases.md:13]: We should remove this.

File: Releases.md [hunk 2]
```diff
@@ -6,6 +6,34 @@ https://github.com/denoland/deno/releases
...
+- Remove unstable Deno.sleepSync (#14719)
```
Comment (dsherret) [inline Releases.md:14]: Maybe add `BREAKING:` and resort so it's up with the other BREAKING.

**Review by aslilac — APPROVED** [review]

**Review by dsherret — APPROVED** [review]
LGTM

File: Releases.md [hunk 3]
```diff
@@ -1489,7 +1516,7 @@ Changes in std version 0.85.0:
 - fix: use inline source maps when present in js (#8995)
 - fix: use tokio for async fs ops (#9042)
 - refactor(cli): remove 'js' module, simplify compiler snapshot (#9020)
-- refactor(op_crates/crypto): Prefix ops with "op_crypto_" (#9067)
+- refactor(op*crates/crypto): Prefix ops with "op_crypto*" (#9067)
```
Comment (dsherret) [inline Releases.md:1519]: Is this intentional? The other escapes look good. We should build that into the script that updates this file.

File: Releases.md [hunk 4]
```diff
@@ -6,6 +6,33 @@ https://github.com/denoland/deno/releases
...
+- feat(repl): Add key binding to force a new line (#14536)
+- feat(runtime/signal): implement SIGINT and SIGBREAK for windows (#14694)
+- feat(task): add `--cwd` flag for configuring the working directory (#14823)
+- feat(task): support redirects, cat, xargs (#14859)
+- feat(test): update test summary report (#14629)
+- feat(vendor): support using an existing import map (#14836)
+- feat: make Child.kill argument optional (#14669)
+- feat: no type-check by default (#14691)
+- feat: update to TypeScript 4.7 (#14242)
+- feature(web): enable deflate-raw compression format (#14863)
```
Comment (dsherret) [inline Releases.md:29]: While you're here, we could also standardize this with `feature` -> `feat`

File: Releases.md (hunk 3 above)
Comment (aslilac) [reply Releases.md:1519]: I think Prettier ran when I saved and I didn't actually look at the diff 🙃

**Pull request merged**